src/EnrollChoicesController.js

```diff
@@ -36,13 +40,11 @@ function (Okta, FormController, Enums, RouterUtil, FactorList,
     Form: {
       noCancelButton: true,
 
-      title: _.partial(Okta.loc, 'enroll.choices.title', 'login'),
-
       noButtonBar: function () {
         return this.state.get('pageType') === Enums.ALL_OPTIONAL_NONE_ENROLLED;
       },
 
-      subtitle: function () {
```

Comment:
same here, are we changing this to add a specific css class to the subtitle?

Reply:
The main reason i need to put this in a template is because the courage template for subtitle uses {{subtitle}} which escapes the html. I'm using triple stash {{{subtitle}}} here